packages/@dataform/core/BUILD

```diff
@@ -99,7 +93,6 @@ pkg_npm_tar(
         ":bundle_with_license",
         ":configs_proto_with_license",
         ":core_proto_with_license",
-        ":data_preparation_proto_with_license",
```

Comment:
You're still going to need this exported, for the decode step

Reply:
I added this back as discussed.